source/documentation/breaking-changes/calc-functions.md

```diff
@@ -0,0 +1,121 @@
+---
+title: 'Breaking Change: Calc functions'
+introduction: >
+  Versions of Dart Sass subsequent to 1.65.0 parse the following as calculation
+  functions: `round()`, `mod()`, `rem()`, `sin()`, `cos()`, `tan()`, `asin()`,
+  `acos()`, `atan()`, `atan2()`, `pow()`, `sqrt()`, `hypot()`, `log()`, `exp()`,
+  `abs()`, and `sign()`.
+--------------------------------------------------------------------------------
+
+These calc functions are now reserved words, which means any custom defined
+functions defined with: [`round()`], [`mod()`], [`rem()`], [`sin()`], [`cos()`], [`tan()`],
+[`asin()`], [`acos()`], [`atan()`], [`atan2()`], [`pow()`], [`sqrt()`], [`hypot()`],
+[`log()`], [`exp()`], [`abs()`], or [`sign()`] will be parsed as calculations
+instead of functions.
+
+[`round()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/round
+[`abs()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/abs
+[`sin()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/sin
+[`cos()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/cos
+[`tan()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/tan
+[`asin()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/asin
+[`acos()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/acos
+[`atan()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/atan
+[`atan2()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/atan2
+[`pow()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/pow
+[`sqrt()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/sqrt
+[`hypot()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/hypot
+[`log()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/log
+[`exp()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/exp
+[`mod()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/mod
+[`rem()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/rem
+[`sign()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/sign
+
+{% codeExample 'calc-function-bad', false %}
+@function rem($number) {
+  @return $number % 10;
+}
+
+/* This example would parse `rem()` as a calc function, not as the declared
+function above. Since `rem()` requires two parameters, this would throw a
+parameter error. */
```

Comment:
You can be more active with the wording of this comment, and following other examples in other breaking change announcements you can add "WRONG" at the beginning to indicate it will cause an error
```
// WRONG, Sass parses `rem()` as a calculation, not as a call to the function
// declared above. This throws an error because `rem()` requires two parameters.
```